mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -215,7 +215,7 @@ message ProcessedTxOut {
     uint64 subaddress_index = 2;
 
     // The public key of the TxOut.
-    external.CompressedRistretto public_key = 3;
+    external.CompressedRistretto tx_public_key = 3;
```

Comment:
Recommend `tx_out_public_key`

Reply:
renamed throughout to `tx_out_public_key`